WooCommerce/Classes/POS/Presentation/PointOfSaleCollectCashView.swift

```diff
@@ -24,33 +25,27 @@ struct PointOfSaleCollectCashView: View {
                                                                                       allowNegativeNumber: false)
 
     var body: some View {
-        VStack(alignment: .center) {
+        VStack(alignment: .center, spacing: conditionalPadding(8)) {
             HStack {
                 Button(action: {
                     Task { @MainActor in
                         await posModel.cancelCashPayment()
                     }
                 }, label: {
-                    HStack(alignment: .top) {
-                        Image(systemName: "chevron.backward")
-                            .font(.posBodyEmphasized, maximumContentSizeCategory: .accessibilityLarge)
-                        VStack(alignment: .leading) {
-                            Text(Localization.backNavigationTitle)
-                                .font(.posTitleEmphasized)
-                                .accessibilityAddTraits(.isHeader)
-
-                            Text(formattedOrderTotal)
-                                .font(.posBodyRegular)
-                        }
-                        .padding(.top, -Constants.navigationButtonSpacing)
-                    }
-                    .foregroundColor(navigationForegroundColor)
+                    navigationHeader
                 })
                 .disabled(isLoading)
                 Spacer()
+                    .renderedIf(!dynamicTypeSize.isAccessibilitySize)
             }
 
             FormattableAmountTextField(viewModel: textFieldViewModel, style: .pos)
+                .scaleEffect(dynamicTypeSize.isAccessibilitySize ? 0.7 : 1.0)
```

Comment:
This modifier can also be used to restrict some views from growing past a defined size. Some views may be large enough by themselves. 
```suggestion
                .dynamicTypeSize(...DynamicTypeSize.accessibility3)
```

Reply:
💯  I didn't know that `...upperbound` syntax. I've updated it to use this instead https://github.com/woocommerce/woocommerce-ios/pull/14935/commits/3d3c077f9d2444eb33b249a9f31270edffacd8b2